styles/styles.scss

```diff
@@ -79,6 +79,7 @@ $footer-background-color: black;
 $footer-color: white;
 
 @import "../node_modules/bulma/bulma.sass";
+@import "../node_modules/bulma-print/bulma-print";
```

Comment:
Apparently, what this import refers to is just [a few lines of CSS](https://github.com/suterma/bulma-print/blob/7389bd6b28798c89163ed8886eaed0162b4aee96/sass/helpers.scss#L6-L40), so I think we can simply paste it inline in `styles/styles.scss` rather than adding bulma-print. What do you think?

(Also, it seems that `bulma-print` library hasn't been updated for some time.)